airbyte-integrations/connectors/source-dremio/integration_tests/sample_state.json

```diff
@@ -0,0 +1,10 @@
+{"data": [{
```

Comment:
why do we need a sample state if the connector does not support incremental syncs?

Reply:
I agree, removed this file